From 6e402d97c2dec5726f37e95f97b7f7e12b1d3b1d Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Wed, 11 Nov 2020 11:02:13 +0100
Subject: [PATCH] IWYU: include headers for std::vector and std::unique_ptr in AXTreeFormatter

Fix these build errors with libstdc++:
../../ui/accessibility/platform/inspect/tree_formatter.h:35:12: error: ‘std::vector’ has not been declared
../../ui/accessibility/platform/inspect/tree_formatter.h:61:16: error: ‘unique_ptr’ in namespace ‘std’ does not name a template type

Bug: 957519
Change-Id: I402ac0644255b3cd4932ff2fe72d999b125a7895
---

diff --git a/ui/accessibility/platform/inspect/tree_formatter.h b/ui/accessibility/platform/inspect/tree_formatter.h
index 4a70a4d..bb23768 100644
--- ui/accessibility/platform/inspect/tree_formatter.h
+++ ui/accessibility/platform/inspect/tree_formatter.h
@@ -5,6 +5,9 @@
 #ifndef UI_ACCESSIBILITY_PLATFORM_INSPECT_TREE_FORMATTER_H_
 #define UI_ACCESSIBILITY_PLATFORM_INSPECT_TREE_FORMATTER_H_
 
+#include <memory>
+#include <vector>
+
 #include "ui/accessibility/platform/inspect/inspect.h"
 
 #include "ui/gfx/native_widget_types.h"
